src/meta/src/manager/catalog.rs

```diff
@@ -335,7 +334,11 @@ where
         }
     }
 
-    pub async fn drop_table(&self, table_id: TableId) -> MetaResult<NotificationVersion> {
+    pub async fn drop_table(
+        &self,
+        table_id: TableId,
+        internal_table_ids: Vec<TableId>,
```

Comment:
I think there should be no internal tables for `TableV2`?

Reply:
Yes, it has. 😂 Table will have several internal tables if it got internal states. These will be stored as 'Table' in catalog manager and synced to frontend. 
Note that: calling 'show materialized views' will filter out internal states.

Reply:
Do you mean that the table here can also ref to a materialized view?

Reply:
Yep, table here is actually a materialized view.